config/PropertyNames.xml

```diff
@@ -411,6 +411,7 @@
         <property name="Node.RetryCount" default="6" languages="cpp" />
         <property name="Node.RetryDelay" default="500" languages="cpp" />
         <property name="Node.RetryMultiplier" default="2" languages="cpp" />
+        <property name="Node.RetryTimeout" default="10000" languages="cpp" />
```

Comment:
A new property that controls how much time we wait for a recovery attempt to success before we schedule a new one. Defaults to 10s.